File: Unit 1 Fundamentals of programming/Fundamentals Topic 1 Programming Basics/Topic 1 Python Programs/W1 Q4 number of books.py
#AQA AS W1 Q4 Number of Books
numberOfStudents = int(input("Enter number of students: "))
numberOfBooks = int(input("Enter number of books: "))
booksPerStudent = numberOfBooks // numberOfStudents
leftOver = numberOfBooks % numberOfStudents
print("Books per student: ",booksPerStudent)
print("Books left over: ", leftOver)

File: A-Level/Unit 1 Fundamentals of programming/Fundamentals Topic 1 Programming Basics/Topic 1 VB Programs/W1 Q4  Number of Books.vb
Module Module1

    Sub Main()
        'W1 Q4 Number of Books
        Dim numberOfBooks As Integer = InputBox("Enter number of books")
        Dim numberOfStudents As Integer = InputBox("Enter number of students")
        Dim numberOfBooksPerStudent As Integer = numberOfBooks / numberOfStudents
        Dim numberLeftOver As Integer = numberOfBooks Mod numberOfStudents


        MsgBox("Number of books per student: " & numberOfBooksPerStudent)
        MsgBox("Books left over: " & numberLeftOver)


    End Sub

End Module
